source/Halibut/Transport/SecureListener.cs

```diff
@@ -143,16 +143,15 @@ void WaitForPendingConnectionOrCancellation()
                         }
 
                         var client = listener.AcceptTcpClient();
-#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
-                        HandleClient(client);
-#pragma warning restore CS4014
+                        Task.Run(async () => await HandleClient(client).ConfigureAwait(false)).ConfigureAwait(false);
```

Comment:
I don't think we need `ConfigureAwait(false)` here, given we are not awaiting

Reply:
Which brings me to untrackable tasks. I think we need to track this dangling tasks

Reply:
Can you clarify what you mean please @johnsimons ?

Reply:
See https://github.com/OctopusDeploy/Halibut/pull/36/files#diff-26fa9bedf8a3cc85fcb9b46910dfe94d21c3a269b5f53a26cdacac8c928f7656R61, there I am tracking the returned task, so I can dispose of it correctly at shutdown time

Reply:
This article says we don't need to: https://devblogs.microsoft.com/pfxteam/do-i-need-to-dispose-of-tasks/#:~:text=medium%2Dlength%20answer%3A-,%E2%80%9CNo.,to%20meet%20your%20performance%20goals.

And stack overflow answer: https://stackoverflow.com/a/3734298/10784

Taub works(ed) on the .NET parallel tasks team apparently